contracts/Commitment/Commitment.sol

```diff
@@ -563,6 +556,13 @@ contract Commitment is
         NEUMARK.distributeNeumark(msg.sender, investorNmk);
         NEUMARK.distributeNeumark(PLATFORM_WALLET, platformNmk);
 
-        return (investorNmk, ticketNmk);
+        if (ticketNmk > 0) {
```

Comment:
Maybe it's better to move this logic inside `if (whitelisted && whitelistActiveForToken) {` block?

Reply:
nice simplification! in 74cd932